ibm_db2/tests/test_unit.py

```diff
@@ -44,3 +46,25 @@ def mock_exception(*args, **kwargs):
             ibmdb2.check(instance)
         # new connection made
         assert ibmdb2._conn != conn1
+
+
+def test_metadata(aggregator, instance, datadog_agent):
+    check = IbmDb2Check('ibm_db2', {}, [instance])
+    check.check_id = CHECK_ID
+
+    check.check(instance)
+
+    # The version string differs from the image tag version
+    version = get_version(check._conn)
```

Comment:
Where is this getting the version from?

Reply:
This is getting the version from the IBM DB2 server https://github.com/ibmdb/python-ibmdb/wiki/APIs#ibm_dbserver_info